scripts/grpc/requirements.txt

```diff
@@ -1 +1 @@
-protobuf>=4.21.12
+protobuf>=3.21.9
```

Comment:
Changing these versions will break gRPC tests for someone again :)
We've only just found a good combination that satisfies most users